### DISCUSSION TITLE
VSCode always saves?

### DISCUSSION THREAD

**Discussion by EduardoVB** [twinbasic/twinbasic#323] [Q&A] [answered]
I made some modifications in a project, just for testing, I deleted a whole section of code, then closed vscode IDE.
I din't get any message for saving or not saving the changes.

When I reopened the project, the last changes were saved and I lost the section that I had deleted.

I'm not sure whether to report it as a bug (by opening an issue) but it does not seem normal to me.

**Comment by WaynePhillipsEA** [comment, marked as answer]
You can turn off the VS Code 'Hot Exit' feature from File menu > Preferences > Settings >  search Hot Exit > set to Off
